plugin/trino-hive-hadoop2/bin/common.sh

```diff
@@ -76,12 +76,30 @@ function cleanup_hadoop_docker_containers() {
     cleanup_docker_containers "${DOCKER_COMPOSE_LOCATION}"
 }
 
-function termination_handler(){
+function termination_handler() {
     set +e
     cleanup_docker_containers "$@"
     exit 130
 }
 
+# Check that all arguments are the names of non-empty variables.
+function require_env() {
+    ( # Use a subshell to avoid errexit and preserve xtrace
+        set +x
+        local failing=0
+        for arg; do
```

Comment:
i don't think we need this to be a `for` loop.
the function can be made to accept single variable name to check

Reply:
I made it this way so it only needs to run once to detect all the missing/empty variables, because it can be annoying to have to run it multiple times to find out which variables you forgot.

But this  isn't very important; should I change it?